README.md

```diff
@@ -69,7 +69,9 @@ Currently, [Azure Key Vault plugin for Notation](https://github.com/Azure/notati
     plugin_url: https://github.com/Azure/notation-azure-kv/releases/download/v1.0.1/notation-azure-kv_1.0.1_linux_amd64.tar.gz
     plugin_checksum: f8a75d9234db90069d9eb5660e5374820edf36d710bd063f4ef81e7063d3810b
     key_id: https://testnotationakv.vault.azure.net/keys/notationLeafCert/c585b8ad8fc542b28e41e555d9b3a1fd
-    target_artifact_reference: myRegistry.azurecr.io/myRepo@sha256:aaabbb
+    target_artifact_reference: |-
```

Comment:
Is this character `|-` necessary when specifying artifact references?

Reply:
No, it's not a MUST, in case you are interested: https://stackoverflow.com/questions/3790454/how-do-i-break-a-string-in-yaml-over-multiple-lines.

Since we've already using it for the `plugin-config` input, I'm just keeping the format here.